README.md

```diff
@@ -74,89 +88,216 @@ spec:
+```
+
+- You can also create `cr-patch.yaml` and use it with chectl:
+
+```yaml
+spec:
+  auth:
+    openShiftoAuth: false
+```
+
+And update che-cluster using chectl:
+
+```
+$ chectl server:update -n ${eclipse-che-namespace} --che-operator-cr-patch-yaml=/path/to/cr-patch.yaml
+```
+
+## TLS
 
 TLS is enabled by default.
 Turning it off is not recommended as it will cause malfunction of some components.
+But for development purposes you can do that.
+
+- using command line:
+
+```bash
+$ kubectl patch checluster/eclipse-che -n ${eclipse-che-namespace} --type=merge -p '{"spec":{"server":{"tlsSupport": false}}}'
+```
+
+- Also you can create `cr-patch.yaml` and use it with chectl:
+
+```yaml
+spec:
+  server:
+    tlsSupport: false
+```
+
+And update che-cluster using chectl:
+
+```
+$ chectl server:update -n ${eclipse-che-namespace} --che-operator-cr-patch-yaml=/path/to/cr-patch.yaml
+```
+
+### Deploy multi-user Che
+
+che-operator deploys Eclipse Che with enabled multi-user mode by default.
+To start work each user should login/register using form, after that user will be redirected to the user dashboard.
+
+### Deploy single user Che
+
+To enable single user Che you can:
+
+- use command line:
+
+```bash
+$ kubectl patch checluster/eclipse-che -n ${eclipse-che-namespace} --type=merge -p '{"spec":{"server": {"customCheProperties": {"CHE_MULTIUSER": "false"}}}}'
+```
+
+- Also you can create `cr-patch.yaml` and use it with chectl:
+
+```yaml
+spec:
+  server:
+    customCheProperties:
+      CHE_MULTIUSER: "false"
+```
+
+And update che-cluster using chectl:
+
+```
+$ chectl server:update -n ${eclipse-che-namespace} --che-operator-cr-patch-yaml=/path/to/cr-patch.yaml
+```
+
+### Deploy Che with single host strategy
+
+todo
 
 #### OpenShift
 
-When the cluster is configured to use self-signed certificates for the router, the certificate will be automatically propogated to Che components as trusted.
+When the cluster is configured to use self-signed certificates for the router, the certificate will be automatically propagated to Che components as trusted.
 If cluster router uses certificate signed by self-signed one, then parent/root CA certificate should be added into corresponding config map of additional trusted certificates (see `serverTrustStoreConfigMapName` option).
 
 #### K8S
 
 By default self-signed certificates for Che will be generated automatically.
-If it is needed to use own certificates, create `che-tls` secret (see `k8s.tlsSecretName` option) with `key.crt` and `tls.crt` fields. In case of self-signed certificate `self-signed-certificate` secret should be created with public part of CA certificate under `ca.crt` key in secret data.
+If it is needed to use your own certificates, create `che-tls` secret (see `k8s.tlsSecretName` option) with `key.crt` and `tls.crt` fields. In case of self-signed certificate `self-signed-certificate` secret should be created with the public part of CA certificate under `ca.crt` key in secret data.
```

Comment:
Here we can add a note, that is is possible to use default certificate of Kubernetes cluster by passing empty string as a value of `tlsSecretName`:
```yaml
spec:
  k8s:
    tlsSecretName: ''
```